src/metrics.rs

```diff
@@ -3004,6 +3030,44 @@ impl GooseAttack {
                 status_code_template = "".to_string();
             }
 
+            let graph_rps_template: String;
+            if Some(starting) == self.metrics.starting
+                && Some(started) == self.metrics.started
+                && Some(stopping) == self.metrics.stopping
+                && Some(stopped) == self.metrics.stopped
+            {
+                let mut count = 0;
+                for (_path, path_metric) in self.metrics.requests.iter() {
```

Comment:
If `--report-file` is enabled and `--no-reset-metrics` is not enabled, we should store a copy of the `metrics.requests` "somewhere else" before we reset the metrics so we can generate a complete chart. Alternatively, in this configuration perhaps we should simply not display the "Starting" portion of the chart as it will always be blank.